sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala

```diff
@@ -53,31 +53,31 @@ private[sql] class JSONOptions(
   val samplingRatio =
     parameters.get("samplingRatio").map(_.toDouble).getOrElse(1.0)
   val primitivesAsString =
-    parameters.get("primitivesAsString").map(_.toBoolean).getOrElse(false)
+    parameters.get("primitivesAsString").exists(_.toBoolean)
```

Comment:
I know this calls less but I wonder if it is really more readable. To me `map(_).getOrElse(false)` looks more readable: if exists convert to a bool, and defaults `false`.

Reply:
Got it ~ I will close this pr ~ thx ~